setup.py

```diff
@@ -3,9 +3,7 @@
 from setuptools import setup
 import re, os
 
-version = ''
-with open('koreanbots/__init__.py', encoding='UTF8') as f:
-    version = re.search(r'^__version__\s*=\s*[\'"]([^\'"]*)[\'"]', f.read(), re.MULTILINE).group(1)
+version = '2.0.0'
```

Comment:
Save version on __init__.py for version information to be accessible on inline code.